### ISSUE TITLE
Update ConsentBanner component to use updated designs

### ISSUE THREAD

**Issue by andrewscfc** [BBC-archive/psammead#4216]
**Is your feature request related to a problem? Please describe.**
We have updated the consent banner designs as shown here: https://zpl.io/bzKA46A, we need to apply to our amp and canonical consent banners.

Amp extended functionality will be covered by a seperate issue #4215 

We also need to address long standing accessibility issues, the accessibility criteria is below:
<details>
<summary>Accessibility Criteria</summary>

> Content Order 
> 
> Scenario: Content Order 
> 
> - Given I use a screen reader
> - When I navigate to the privacy or consent banner 
> - Then this should be the first content on the page
> - And the content order of the banner should match the screen reader UX

> Landmarks
> 
> Scenario: Landmarks 
> 
> - Given I use a screen reader
> - When I navigate to the consent or privacy banner on a canonical page  
> - Then a banner landmark will be announced
> - And this should not be nested within another landmark e.g. the main landmark or another banner landmark
> 
> Focus management
> 
> Scenario: Focus 
> 
> - Given I have activated the “OK” button on the privacy banner  
> - When the cookie banner is then displayed
> - Then focus should be on the H2 at the top of banner e.g. (“Let us know you agree to Cookies”) 
> 
> Scenario: Focus from buttons 
> 
> - Given I am on a consent banner 
> - When I select to accept/reject
> - Then the consent banner is closed  
> - And my focus is returned to the top of the page
> 
> Buttons/Links
> 
> - Given I am viewing [component]
> - When I inspect the code
> - Then ‘OK/Yes, I accept’ is a button HTML element
> - And not a link HTML element

> - Given I am viewing [component]
> - When I inspect the code
> - Then the ‘No, take me to settings/Find out what has changed’ is a link HTML element
> - And not a button HTML element

> - Given I am viewing [component]
> - And I tap on all buttons
> - Then touch targets must be large enough to touch accurately 
> - And they must have a width and height of 44px as a minimum
> 
> Scenario: Links/Buttons - Hover 
> 
> - Given I use a mouse
> - When I hover over link/button elements
> - Then they have a visible state change
> - And this is not identified by colour alone
> 
> Scenario: Links/Buttons - Focus
> 
> - Given I use a keyboard only
> - When I navigate through [component] by pressing the TAB key
> - Then all links/buttons elements are focusable
> - And they have visible state change on focus 
> - And this is not identified by colour alone
> - And this is in addition to the browser default focus outline styling
> 
> Colour contrast
> 
> Scenario: Paragraph text - colour contrast 
> 
> - Given I am viewing the consent banner on amp
> - When I view the paragraph text 
> - Then the number must meet the colour contrast ratio of 4.1:5 or higher
> 
> User preferences
> 
> Scenario: Changing colours
>
> - Given I am using Windows High Contrast mode
> - And also when I change my colour preferences in FireFox
> - When I view the consent/privacy banner
> - Then an outline is visible for all buttons
> - And I can see where the top of the compoment is e.g. there is a line/border

> - Given I am using Windows High Contrast mode
> - And also when I change my colour preferences in FireFox
> - When I hover over link elements
> - Then they have a visible state change

>- Given I am using Windows High Contrast mode
>- And also when I change my colour preferences in FireFox
>- When I navigate through [component] by pressing the TAB key
>- Then all links have a visible state change
>- And this is in addition to the browser default focus outline styling

</details>

Screen reader UX can be found here: https://zpl.io/2y4zlLJ

**Describe the solution you'd like**
Refer to designs above and update the component: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-consent-banner/src/index.jsx

**Describe alternatives you've considered**

**Testing notes**
- Can be tested when integrated into Simorgh
- Update snapshots reflecting new design
- Update chromatic snapshots to reflect new design

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by simonsinclair** [comment]
This issue was resolved by #4225.
For follow-on work, please see https://github.com/bbc/psammead/issues/4290.
